src/main/java/juja/microservices/gamification/service/AchievementService.java

```diff
@@ -87,6 +88,52 @@
         return Collections.singletonList(id);
     }
 
+    public List<String> addAdmin(AdminAddRequest request) {
+        List<String> result = new ArrayList<>();
+        AchievementType type = request.getType();
+
+        logger.debug("Received request on /achievements/{} : {}", type.toString(), request);
+
```

Comment:
why do you write to logger "Received request on /achievements/{type}" 
if request has daily type, that the url will be /achievements/daily
but actual url was /achievements/admin ?
I think better - ("received admin achievement: {}", request)

Reply:
agree, could be confusing, changed.